apps/charterafrica/src/payload/globals/Helpdesk.js

```diff
@@ -42,7 +43,7 @@ const Helpdesk = {
           admin: {
             elements: ["ol", "ul", "link"],
           },
```

Comment:
I handled this [here](https://github.com/CodeForAfrica/ui/blob/90bef53e1062b1d016e07f8b64ff036cd2f17dff/apps/charterafrica/src/payload/fields/richText.js#L48)

Reply:
🤔 ... cool hack but unless there is a **significant** number of places we need to change, I'd rather we move `admin` into `editor` everywhere.

Otherwise, we'd need to maintain two versions of code moving forward.

Reply:
🤔 we also do not need to be importing slate editor everywhere richText function is imported.
Perhaps maybe rename  admin to editor would be ideal.